src/domain/courseSearch/utils.ts

```diff
@@ -0,0 +1,78 @@
+import { forEach, isArray, isEmpty, isNil, isNumber } from 'lodash';
+
+import { Meta } from '../../generated/graphql';
+import { formatDate } from '../../util/dateUtils';
+import getUrlParamAsArray from '../../util/getUrlParamAsArray';
+import { EVENT_SEARCH_FILTERS } from '../eventSearch/constants';
+import { Filters, MappedFilters } from './types';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const getSearchFilters = (searchParams: URLSearchParams) => {
```

Comment:
My IDE says that none of these methods are never used anywhere in the codebase. Accidental or is this code that will be used in the future? No test coverage either.

Reply:
Hmm, I think that this might have already been deleted in some commit but came back in some rebase or merge... not sure

Anyway, now deleted